### PR TITLE
Add `s3_bucket_name` and `s3_bucket_path` to `cluster_info`

### PR DESCRIPTION
**Pull request by rnetser**
When destroying clusters from directories, `s3_bucket_name` and `s3_bucket_path` should be taken from cluster_data.yaml file and not from cli

### CODE REVIEW

**Review by myakove — APPROVED** [review]

File: openshift_cli_installer/utils/clusters.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ def clusters_from_directories(directories):
                         _data = yaml.safe_load(fd)
 
                     _data["cluster_info"]["cluster-dir"] = root
+                    _data["cluster_info"]["s3_bucket_name"] = _data.get("s3_bucket_name")
```
Comment (myakove) [inline openshift_cli_installer/utils/clusters.py:40]: I don't understand why this info is needed.

Comment (rnetser) [reply openshift_cli_installer/utils/clusters.py:40]: updated the description

**Pull request merged**